src/main/java/com/zendesk/maxwell/replay/ReplayConfig.java

```diff
@@ -0,0 +1,101 @@
+package com.zendesk.maxwell.replay;
+
+import com.zendesk.maxwell.MaxwellConfig;
+import com.zendesk.maxwell.util.MaxwellOptionParser;
+import joptsimple.OptionSet;
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+
+/**
+ * @author udyr@shlaji.com
+ */
+public class ReplayConfig extends MaxwellConfig {
+	private static final Logger LOGGER = LoggerFactory.getLogger(MaxwellReplayFile.class);
+
+	List<ReplayFilePattern> binlogFiles = new ArrayList<>();
+
+	public ReplayConfig(String[] args) {
+		super(args);
+		this.parse(args);
+		replayMode = true;
+	}
+
+	private void parse(String[] args) {
+		MaxwellOptionParser parser = buildOptionParser();
+		OptionSet options = parser.parse(args);
+
+		final Properties properties;
+
+		if (options.has("config")) {
+			properties = parseFile((String) options.valueOf("config"), true);
+		} else {
+			properties = parseFile(DEFAULT_CONFIG_FILE, false);
+		}
+
+		if (options.has("env_config")) {
+			Properties envConfigProperties = readPropertiesEnv((String) options.valueOf("env_config"));
+			for (Map.Entry<Object, Object> entry : envConfigProperties.entrySet()) {
+				Object key = entry.getKey();
+				if (properties.put(key, entry.getValue()) != null) {
+					LOGGER.debug("Replaced config key {} with value from env_config", key);
+				}
+			}
+		}
+
+		String envConfigPrefix = fetchStringOption("env_config_prefix", options, properties, null);
+
+		if (envConfigPrefix != null) {
+			String prefix = envConfigPrefix.toLowerCase();
+			System.getenv().entrySet().stream()
+					.filter(map -> map.getKey().toLowerCase().startsWith(prefix))
+					.forEach(config -> {
+						String rawKey = config.getKey();
+						String newKey = rawKey.toLowerCase().replaceFirst(prefix, "");
+						if (properties.put(newKey, config.getValue()) != null) {
+							LOGGER.debug("Got env variable {} and replacing config key {}", rawKey, newKey);
+						} else {
+							LOGGER.debug("Got env variable {} as config key {}", rawKey, newKey);
+						}
+					});
+		}
+
+		String replayBinlog = fetchStringOption("replay_binlog", options, properties, null);
+		if (StringUtils.isBlank(replayBinlog)) {
+			usage("Please specify a replay_binlog: ", parser, "replay");
+		}
+		this.binlogFiles.addAll(validateReplayFiles(replayBinlog, parser));
+
+	}
+
+	private List<ReplayFilePattern> validateReplayFiles(String replayBinlog, MaxwellOptionParser parser) {
+		List<ReplayFilePattern> filePatterns = FilePathParser.parse(replayBinlog);
```

Comment:
yeah, don't do any of this parsing -- just take a list of binlog files as arguments.

Reply:
to be clear, invoking `maxwell-replay` should look like:

```
bin/maxwell-replay --host=XXX --user=YYY /var/log/binlogfile_1 /var/log/binlogfile_2 /tmp/binlogfile_3
```

Reply:
I actually have a lot of binlog files, and I need to select dozens of files from hundreds of files, so I want to solve this problem with a regular expression.

`binlog.000001,binlog.000002,...,binlog.000233,....,binlog.000453`

I mount over 400 files from cloud disk in my directory

Through the binlog **position** I finally confirm that I need to replay  binlog.000233 ~ binlog.000289

Reply:
mmmm no.   Solve this problem outside of maxwell; parse your binlog position (in bash or whatever) and then filter your file list via other methods.  Then pass them all in as command line arguments.  The weird regex stuff is too particular to your specific use case.

Reply:
this is fine, I need a script to generate a long file configuration, I will provide a helper script